macros/print_base/print_start.cfg

```diff
@@ -153,14 +155,16 @@ gcode:
     # Set vars
     {% set St = printer["gcode_macro _USER_VARIABLES"].travel_speed * 60 %}
     {% set Sz = printer["gcode_macro _USER_VARIABLES"].z_drop_speed * 60 %}
+    {% set prime_line_x = printer["gcode_macro _USER_VARIABLES"].prime_line_x %}
+    
 
     G91
     M83
     G1 Z5 F{Sz}
 
     ; Starting position
     G90
-    G1 X2.5 Y20 F{St}
+    G1 X{prime_line_x} Y20 F{St}
```

Comment:
Is it possible to also put the Y in the variables file ?

like X,Y start position or something. If possible it could be better to keep them on only one variable like :
```
prime_line_start: 3,20
```

And in the macro:
```
{% set start_pos_x, start_pos_y = printer["gcode_macro _USER_VARIABLES"].prime_line_start.split(',')|map('trim')|map('float') %}
```